docs/resources/variable.md

```diff
@@ -44,8 +44,8 @@ resource "prefect_variable" "example" {
 Import is supported using the following syntax:
 
 ```shell
-# Prefect Variables can be imported via name in the form `name/name-of-variable`
-terraform import prefect_variable.example name/name-of-variable
+# Prefect Variables can be imported via name in the form `name/name_of_variable`
+terraform import prefect_variable.example name/name_of_variable
```

Comment:
does it not like hyphens?

Reply:
correct - lower case and underscores only

Reply:
ah wild, TIL